src/enact/registration.py

```diff
@@ -78,11 +78,11 @@ def register(registerable: RegisterableT) -> RegisterableT:
   elif (isinstance(registerable, type) and
       issubclass(registerable, interfaces.ResourceBase)):
     return cast(RegisterableT, resource_registry.register(registerable))
+  elif isinstance(registerable, type):
+    raise TypeError(
+      f'Cannot register type {registerable}. '
+      f'Please inherit from Resource or Invokable to register types.')
```

Comment:
I think "to register types" is a bit unclear. 

```
Cannot ... Only types that inherit from Resource or Invokable can be registered.
Consider using a ResourceWrapper to register non-Resource types.
```

Reply:
Fixed.